app/code/Magento/SalesRule/Test/Mftf/Test/CartPriceRuleForBundleProductTest.xml

```diff
@@ -0,0 +1,159 @@
+            <actionGroup ref="NavigateToEditProductAttributeActionGroup" stepKey="editSkuAttribute">
+                <argument name="ProductAttribute" value="sku" />
+            </actionGroup>
+            <actionGroup ref="ChangeUseForPromoRuleConditionsProductAttributeActionGroup" stepKey="changeAttributePromoRule">
+                <argument name="option" value="1" />
+            </actionGroup>
+
+            <!-- Run reindex stock status -->
+            <magentoCLI command="cron:run --group=index" stepKey="runCronIndexer"/>
+        </before>
+
+        <after>
+            <!-- Delete created SalesRule -->
+            <actionGroup ref="DeleteCartPriceRuleByName" stepKey="DeleteCartPriceRuleByName">
+                <argument name="ruleName" value="{{SimpleSalesRule.name}}"/>
+            </actionGroup>
+
+            <!-- Delete Bundle product and it's children -->
+            <deleteData createDataKey="createBundleProduct" stepKey="createBundleProduct" />
+            <deleteData createDataKey="simpleProduct1" stepKey="simpleProduct1" />
+            <deleteData createDataKey="simpleProduct2" stepKey="simpleProduct2" />
+            <deleteData createDataKey="simpleProduct3" stepKey="simpleProduct3" />
+            <deleteData createDataKey="simpleProduct4" stepKey="simpleProduct4" />
+
+            <!-- Revert Attribute 'sku' to it's default value (not accessible for Promo Rule Conditions) -->
+            <actionGroup ref="NavigateToEditProductAttributeActionGroup" stepKey="editSkuAttribute">
+                <argument name="ProductAttribute" value="sku" />
+            </actionGroup>
+            <actionGroup ref="ChangeUseForPromoRuleConditionsProductAttributeActionGroup" stepKey="changeAttributePromoRule">
+                <argument name="option" value="0" />
+            </actionGroup>
+
+	        <actionGroup ref="AdminLogoutActionGroup" stepKey="logout"/>
+
+            <!-- Reindex invalidated indices after product attribute has been created/deleted -->
+            <magentoCron groups="index" stepKey="reindexInvalidatedIndices2" />
+        </after>
+
+        <!-- Create the rule -->
+        <amOnPage url="{{AdminCartPriceRulesPage.url}}" stepKey="amOnCartPriceList"/>
+        <waitForPageLoad stepKey="waitForRulesPage"/>
+        <click selector="{{AdminCartPriceRulesSection.addNewRuleButton}}" stepKey="clickAddNewRule"/>
+        <fillField selector="{{AdminCartPriceRulesFormSection.ruleName}}" userInput="{{SimpleSalesRule.name}}" stepKey="fillRuleName"/>
+        <selectOption selector="{{AdminCartPriceRulesFormSection.websites}}" userInput="Main Website" stepKey="selectWebsites"/>
+        <actionGroup ref="SelectNotLoggedInCustomerGroupActionGroup" stepKey="selectNotLoggedInCustomerGroup"/>
+        <generateDate date="-1 day" format="m/d/Y" stepKey="yesterdayDate"/>
+        <fillField selector="{{AdminCartPriceRulesFormSection.fromDate}}" userInput="{$yesterdayDate}" stepKey="fillFromDate"/>
```

Comment:
In the 1st time, it failed here. Don't know why. Same code presents in other 8-9 tests.